### ISSUE TITLE
Use request intercept with with connect() on an existing Chromium instance

### ISSUE THREAD

**Issue by steve-the-crab** [mattsse/chromiumoxide#146]
Hi, thanks for your work on this. It's been really nice to work with so far.

Is it possible to do request interception with an existing Chromium instance do you know?

I have made a naive attempt to get this working by copying the code base into my workspace and modifying the `connect` function in `browser.rs` such that it takes a `BrowserConfig` and replaces `config: None` on line 67 with the supplied config, but the result is that the example code for interception appears to be unsuccessful.

Might there be a way to achieve this?

Thanks!

**Comment by j-mendez** [comment]
@steve-the-crab yes, use ` Browser::connect_with_config` and pass in `request_intercept`.
